### ISSUE TITLE
Impossible to rebuild NI out-of-tree Drivers with DKMS

### ISSUE THREAD

**Issue by iamrubeng** [ni/nilrt#219]
I've compiled latest kernel from [here](https://github.com/ni/linux.git) using latest toolchain from [here](https://www.ni.com/es-es/support/downloads/software-products/download.gnu-c---c---compile-tools-x64.html#477803)

When I try to rebuild NI drivers with DKMS, **make prepare** throws an error: "**glibc 2.34 is not found**"
Is there an option to update glibc?

**Comment by amstewart** [comment]
Hi @rgomezceis,

Could you provide some more detail about your reproducing case? 

I see that you're installing the 5.15 kernel. But what version of the NILRT distribution are you using (`cat /etc/os-release`)?

Am I correct to assume that you have installed the built 5.15 kernel to your NILRT instance and are running something like `dkms autoinstall`, and that's where you're getting the `make prepare` error? Could you attach some logs of the error to provide context?

**Comment by iamrubeng** [comment]
Hello, here the info:

```
ID=nilrt
NAME="NI Linux Real-Time"
VERSION="9.2 (hardknott)"
VERSION_ID=9.2
PRETTY_NAME="NI Linux Real-Time 9.2 (hardknott)"
DISTRO_CODENAME="hardknott"
BUILD_ID="23.0.0f158-x64"
VERSION_CODENAME="hardknott"
```

And here the error after executing 'make prepare' command:

```
admin@NI-cRIO-9045-01F88633:/lib/modules/5.15.92-rt57-g414876561405-dirty/build# make prepare                                                                                                                                                  SYNC    include/config/auto.conf.cmd
scripts/kconfig/conf: /lib/libc.so.6: version `GLIBC_2.34' not found (required by scripts/kconfig/conf)
make[2]: *** [scripts/kconfig/Makefile:77: syncconfig] Error 1
make[1]: *** [Makefile:620: syncconfig] Error 2
make: *** [Makefile:740: include/config/auto.conf.cmd] Error 2
```

I'm following this guide: https://github.com/ni/nilrt/blob/nilrt/master/hardknott/docs/README.kernel.md
Are toolchains using glibc 2.34? NI Linux RT doesn't have this version.

**Comment by iamrubeng** [comment]
Hello @amstewart,

I've tried to compile it using 2018-2019 toolchains and same error.

My host machine is Ubuntu 22.04

**Comment by iamrubeng** [comment]
Resolved using Ubuntu 20.04 that uses glibc 2.31
